element_miniscope/miniscope.py

```diff
@@ -311,6 +346,13 @@ def make(self, key):
 
 @schema
 class ProcessingMethod(dj.Lookup):
+    """Method or analysis software to process miniscope acquisition.
+
+    Attributes:
+        processing_method (foreign key, char16): Recording processing method (e.g. CaImAn).
```

Comment:
Please change all references from `(char##)` to `( varchar(##) )`

Reply:
Sorry - when I saw the latest commit, I thought 'why didn't both `char` references get changed. It's because that's the actual type in this element, counter to the trend of us using varchar everywhere else. I just assumed it was varchar in the element. We should report it as such in the attribute tables with parens: `(foreign key, char(16) )`